src/media/js/main.js

```diff
@@ -215,6 +180,10 @@ require([
             }
         }
 
+        require(['update_banner'], function(update_banner) {
```

Comment:
@ngokevin your opinion on this ? I thought I was clever by trying to get rid of direct dependencies in the main module and load this asynchronously, but it complicates testing as I can no longer rely on `waitForPageLoaded` in casper tests to test this code.